http_server.go

```diff
@@ -77,6 +80,16 @@ func NewServer(l *slog.Logger, ar authenticator.Request, lister NamespaceLister)
 	}
 }
 
+func (s *NamespaceListerServer) WithTLS(enableTLS bool) *NamespaceListerServer {
+	s.useTls = enableTLS
+	return s
+}
+
```

Comment:
```suggestion
```

Reply:
Fixed in f9ea3c6.

Reply:
Actually, I think I'm going to revert this change and leave `WithTLS` in.  I want to be able to disable TLS, even if someone provides private keys.